css-four-column-layout/styles.css

```diff
@@ -0,0 +1,76 @@
+* {
+  font-family: Barlow, serif;
+}
+
+div.column-quarter {
+  text-align: center;
+  display: block;
+  display: flex:;
+  justify-content: center;
+  margin-left: 18px;
+  margin-right: 18px;
+}
+
+.container {
+  width: 1300px;
+  margin: auto;
+}
+
+.page-height {
+  height: 630px;
+}
+
+h2 {
+  margin-top: 5px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-quarter {
+  flex-basis: 25%;
+}
+
```

Comment:
You already have a rule for this class up above in the stylesheet, why do you have two rules, one that specifies that it must be a `div`?